README.md

```diff
@@ -20,4 +20,5 @@ limitations under the License.
 
 This repository was developed as a live demo for the OSESM lecture at TU Wien, spring 2021.
 
-This repository has an `add()` function.
\ No newline at end of file
+This repository has an `add()` function.
+This repository has an `sub()` function.
```

Comment:
Please rewrite this to proper English (`add()` and `sub()` functions) rather than two sentences.

Reply:
Thanks